glide.yaml

```diff
@@ -19,6 +19,6 @@ import:
 - package: github.com/termie/go-shutil
 - package: github.com/mitchellh/go-ps
 - package: github.com/projectcalico/libcalico-go
-  version: 1.0.0-beta
```

Comment:
For master is it not better to remove the version lock completely so that we can easily regenerate the lock file without needing to look up the latest release?